pkg/object/objects.go

```diff
@@ -480,6 +483,27 @@ func manifestDiff(aom, bom map[string]*K8sObject) (string, error) {
 	return sb.String(), nil
 }
 
+func objectIgnorePaths(objectName string, ignoreResources string) []string {
+	var ignorePaths []string
+	for _, ignored := range strings.Split(ignoreResources, ",") {
+		split := strings.Split(ignored, ":")
+		if len(split) < 4 {
+			continue
+		}
+		kind, namespace, name, path := split[0], split[1], split[2], split[3]
```

Comment:
If we want to ignore multiple paths in one resource, we need to add something like this: `<kind>:<ns>:<name>:<pathA>,<kind>:<ns>:<name>:<pathB>` in ignore list, isn't it?

Reply:
Yes, this is a straightforward representation of a path, especially as input from CLI.

Also, I will add RegEx support in path matching for paths like "*.checksum/config-volume"